### PR TITLE
Bump Python dependencies

### LINKED ISSUE #2461: Bump Python dependencies
### Component(s)

operator

### Describe the issue you're reporting

We should bump the auto-instrumentation python dependencies.

### PR DESCRIPTION
**Pull request by yuriolisa**
**Description:** <Describe what has changed.>

Bump Python dependencies to 1.22.0/0.43b0
**Link to tracking Issue:** <Issue number if applicable>
Resolves #2461 
**Testing:** <Describe what testing was performed and which tests were added.>

**Documentation:** <Describe the documentation added.>

### CODE REVIEW

**Review by frzifus — APPROVED** [review]

File: autoinstrumentation/python/requirements.txt [hunk 1]
```diff
@@ -1,56 +1,56 @@
...
-opentelemetry-instrumentation-sqlite3==0.41b0
-opentelemetry-instrumentation-starlette==0.41b0
-opentelemetry-instrumentation-system-metrics==0.41b0
-opentelemetry-instrumentation-tornado==0.41b0
-opentelemetry-instrumentation-tortoiseorm==0.41b0
-opentelemetry-instrumentation-urllib==0.41b0
-opentelemetry-instrumentation-urllib3==0.41b0
-opentelemetry-instrumentation-wsgi==0.41b0
+opentelemetry-instrumentation-aio-pika==0.43b0
+opentelemetry-instrumentation-aiohttp-client==0.43b0
```
Comment (TylerHelmuth) [inline autoinstrumentation/python/requirements.txt:16]: Release [0.42b0](https://github.com/open-telemetry/opentelemetry-python-contrib/releases/tag/v0.42b0) added a new package that we should include: opentelemetry-instrumentation-aiohttp-server

Comment (yuriolisa) [reply autoinstrumentation/python/requirements.txt:16]: Just added it.

**Review by TylerHelmuth — APPROVED** [review]

File: autoinstrumentation/python/requirements.txt (hunk 1 above)
Comment (TylerHelmuth) [reply autoinstrumentation/python/requirements.txt:16]: It looks like it didn't get tagged properly: https://github.com/open-telemetry/opentelemetry-python-contrib/tree/v0.43b0/instrumentation.  And it doesnt show up on https://pypi.org/project/opentelemetry-instrumentation-aiohttp-server/.  So lets skip for now.

Comment (TylerHelmuth) [reply autoinstrumentation/python/requirements.txt:16]: https://github.com/open-telemetry/opentelemetry-python-contrib/issues/2053

Comment (TylerHelmuth) [reply autoinstrumentation/python/requirements.txt:16]: Lets remove this package from our requirements.txt again.  Hopefully they fix the tagging in the future.

**Review by jaronoff97 — APPROVED** [review]

**Pull request merged**